ci-operator/step-registry/osd-ccs/fleet-manager/osd-ccs-fleet-manager-workflow.yaml

```diff
@@ -11,10 +11,11 @@ workflow:
       ROSA_LOGIN_ENV: "integration"
     pre:
     - ref: osd-ccs-fleet-manager-sc-provision
+    - ref: osd-ccs-fleet-manager-test
```

Comment:
It is acceptable that the test step includes the validation, but it is not good to do testing in the workflow(e.g.. add/remove labels). It is better to put the test into the `test` step so that the workflow is not blocked by a testing scenario.

Reply:
@yasun1 Yes. You are right! But the test step is for hosted cluster actually. There won't be too many FM test cases like this and I think the lightweight solution is to use a ref to group a few FM cases here. How about change its name to osd-ccs-fleet-manager-healthcheck?

Reply:
@yasun1 - I have changed that. please let me know if that's what you had in mind. @jianping-shu - can you take another look at my recent changes as well please?